src/stories/Users/VerifyAttributeWithLink/story.js

```diff
@@ -0,0 +1,51 @@
+const usersRepo = requireRepo("users");
+
+const prepare = ({ reqQuery, reqBody, reqParams, req }) => {
+	const payload = {
```

Comment:
@praveen-betalectic This is not correct.

I understand this might feel like a fast and shortcut to do this, but when some developers read this, they won't understand what data you are taking from req object. 

That's the reason in every other story, we use `findKeysFromRequest`, please modify this code.

Reply:
@praveen-betalectic Where did you get this idea to build payload like this, don't you think we would have done like that in every story, I would want to know your opinion/guess why we use findKeysFromRequest but not directly "req" object, we don't even need reqParams etc., right?

Reply:
@rjvim no, that's only because i didn't think about it, so I took the opertinuty  and went through the code, and made the changes as you instructed. and added if condition to handle in case the callback URLs not provided, please review